core/field_multilineinput.js

```diff
@@ -15,18 +15,18 @@
 goog.module('Blockly.FieldMultilineInput');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.Css');
-goog.require('Blockly.Field');
-goog.require('Blockly.fieldRegistry');
-goog.require('Blockly.FieldTextInput');
-goog.require('Blockly.utils');
-goog.require('Blockly.utils.aria');
-goog.require('Blockly.utils.dom');
-goog.require('Blockly.utils.KeyCodes');
-goog.require('Blockly.utils.object');
-goog.require('Blockly.utils.Svg');
-goog.require('Blockly.utils.userAgent');
-goog.require('Blockly.WidgetDiv');
+const Css = goog.require('Blockly.Css');
+const Field = goog.require('Blockly.Field');
+const fieldRegistry = goog.require('Blockly.fieldRegistry');
+const FieldTextInput = goog.require('Blockly.FieldTextInput');
+const KeyCodes = goog.require('Blockly.utils.KeyCodes');
+const {addClass, createSvgElement, getFastTextWidth, getTextWidth, removeClass} = goog.require('Blockly.utils.dom');
+const {DIV} = goog.require('Blockly.WidgetDiv');
+const {G, TEXT} = goog.require('Blockly.utils.Svg');
+const {GECKO} = goog.require('Blockly.utils.userAgent');
```

Comment:
I think on one of @gonfunko 's previous PRs they decided not to destruccture userAgent because the the const nmes can be confusing.